flood_forecast/basic/linear_regression.py

```diff
@@ -53,12 +54,19 @@ def simple_decode(model: Type[torch.nn.Module],
     real_target2 = real_target.clone()
     # Use last value
     ys = src[:, -1, :].unsqueeze(unsqueeze_dim)
+    ys_std_dev = []
     for i in range(0, max_seq_len, output_len):
         with torch.no_grad():
             if meta_data:
                 out = model(src, meta_data)
             else:
-                out = model(src)
+                if probabilistic:
```

Comment:
So here I'm wondering wouldn't it technically be possible to have both a probabilistic model and a model with meta-data. This could probably be addressed in a separate PR though if you want.

Reply:
Hey, yes it would definitely be possible. Making a pr for this now.